src/sourcemaps/tools/sentry-cli.ts

```diff
@@ -0,0 +1,120 @@
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import * as path from 'path';
+import * as fs from 'fs';
+import {
+  abortIfCancelled,
+  addSentryCliRc,
+  getPackageDotJson,
+  hasPackageInstalled,
+  installPackage,
+} from '../../utils/clack-utils';
+
+import { SourceMapUploadToolConfigurationFunction } from './types';
+
+export const configureSentryCLI: SourceMapUploadToolConfigurationFunction =
+  async (options) => {
+    const packageDotJson = await getPackageDotJson();
+
+    await installPackage({
+      packageName: '@sentry/cli',
+      alreadyInstalled: hasPackageInstalled('@sentry/cli', packageDotJson),
+    });
+
+    let validPath = false;
+    let relativeArtifactPath;
+    do {
+      const rawArtifactPath = await abortIfCancelled(
+        clack.text({
+          message:
+            'Provide a path to the folder that contains your build artifacts:',
```

Comment:
m: Should we rephrase this as a question? 

For example
```suggestion
            'Where are your build artifacts located?',
```